package.json

```diff
@@ -8,7 +8,10 @@
 	"scripts": {
 		"build": "rollup -c tools/rollup.module.js && rollup -c tools/rollup.browser.js && rollup -c tools/rollup.browser.min.js",
 		"prepare": "npm test && rm -rf build && npm run build",
-		"test": "eslint src/** tools/** && tslint -c tslint.json types/**.ts"
+		"test": "eslint src/** tools/** && tslint -c tslint.json types/**.ts",
+		"dev": "npm-run-all --parallel watch serve",
+		"watch": "rollup -w -c tools/rollup.browser.min.js",
```

Comment:
No need to use minified version because this is just for development. Uglifying JS code will cost time.

Reply:
@g-plane True, I dd not think about that.